src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -115,7 +115,9 @@ internal partial class CSharpIntroduceVariableService
                 ? SyntaxFactory.Block(declarationStatement)
                 : SyntaxFactory.Block(declarationStatement, SyntaxFactory.ReturnStatement(rewrittenBody));
 
-            newBody = newBody.WithAdditionalAnnotations(Formatter.Annotation);
+            // Add an elastic newline so that the formatter will place this new lambda body across multiple lines.
```

Comment:
❔ Can you give an example here of what would happen if this change wasn't made?

Reply:
Yup.  Take a look at: https://github.com/dotnet/roslyn/blob/b45885bb6c2fee5d1e996ee88ee88412dcc006a4/src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs#L2349-L2373

Without this change, instead of generating:

```c#
        Func<int, int> f = x =>
        {
            var {|Rename:v|} = x + 1;
            return v;
        };
```

We would instead generate:

```c#
        Func<int, int> f = x => { var {|Rename:v|} = x + 1; return v; };
```